### PR TITLE
Add status mocking to Local Development Guide

### PR DESCRIPTION
**Pull request by BrainBacon**
Added an example `gameserver.yaml` configuration that includes the schema for the `status` object that will be returned during `GameServer()` and `WatchGameServer()` calls.

### CODE REVIEW

File: site/content/en/docs/Guides/Client SDKs/local.md [hunk 1]
```diff
@@ -65,4 +65,43 @@ $ ./sdk-server.linux.amd64 --local -f ../../../examples/simple-udp/gameserver.ya
...
+  address: 127.0.0.1
+  # "nodeName" - the name of the node this gameserver instance runs on
+  nodeName: my-gameserver-node
+```
+
+Example invocation:
+
+```console
+$ curl localhost:59358/gameserver
+{"object_meta":{"name":"gds-example","creation_timestamp":"-62135596800"},"spec":{"health":{}},"status":{"state":"ready","address":"127.0.0.1","ports":[{"name":"game","port":7777}]}}
```
Comment (aLekSer) [inline site/content/en/docs/Guides/Client SDKs/local.md]: Just noticed one strange thing - negative creation timestamp:
`"creation_timestamp":"-62135596800"`

Comment (markmandel) [reply site/content/en/docs/Guides/Client SDKs/local.md]: Probably the zero value.

Comment (aLekSer) [reply site/content/en/docs/Guides/Client SDKs/local.md]: Yes this is start of the year 0001 AC when converted from Linux time to human.

**Comment by markmandel** [comment]
Gentle bump on this, see if we can get the CLA signed?

**Comment by BrainBacon** [comment]
@markmandel Sorry, still waiting on approval from legal. I'll definitely ping this thread again when that happens.

**Comment by markmandel** [comment]
No worries. I'll just occasionally ping you :smile: make sure it doesn't drop off the radar!

**Comment by markmandel** [comment]
Time for a regular check in :smile:

**Comment by BrainBacon** [comment]
@markmandel Still waiting on approval. I'll message you with more details.

**Comment by markmandel** [comment]
Back again for a gentle bump.

**Comment by markmandel** [comment]
Where have we ended up in this?

Also, how relevant is it still, given the changes we've recently made to the local tooling?

**Comment by markmandel** [comment]
Any update on this, otherwise I might just close this for now.

**Comment by markmandel** [comment]
I'm going to clean things up, and close this for now. If you end up getting legal approval, please feel free to reopen :+1:

**Pull request closed without merging**